### PR TITLE
feat: searchable certification requests

### PR DESCRIPTION
**Pull request by dleard**

### CODE REVIEW

File: app/containers/Certifier/CertificationRequestsContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+            <td>{facility}</td>
+            <td>{organisation}</td>
+            <td>{status}</td>
+            <td>{certifierName}</td>
+            <td>{formatListViewDate(node.certifiedAt)}</td>
+            <td>
+              <Link
+                href={`/certifier/certify?applicationId=${applicationId}&version=${versionNumber}`}
+              >
+                <Button className="w-100">View</Button>
```
Comment (kriscooke) [inline app/containers/Certifier/CertificationRequestsContainer.tsx:86]: This `applicationId` appears to be the "short" id, whereas you need the "long" one in order to construct a valid URL to the certify page. Can you get that from your search function? Try clicking on the link here.

File: app/containers/Certifier/CertificationRequestsContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+  const maxResultsPerPage = 20;
+
+  const maxPages = Math.ceil(
+    query?.searchCertificationRequests?.edges[0]?.node?.totalRequestCount /
+      maxResultsPerPage
+  );
+
+  return query.searchCertificationRequests.edges.length === 0 ? (
+    <Alert variant="info">You have no current certification requests.</Alert>
```
Comment (matthieu-foucault) [inline app/containers/Certifier/CertificationRequestsContainer.tsx:100]: Should it be "You have no current certification requests matching your search criteria"?

File: app/containers/Certifier/CertificationRequestsContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,183 @@
+import React, {useEffect, useState} from 'react';
+import {Button, Alert} from 'react-bootstrap';
+import moment from 'moment-timezone';
+import Link from 'next/link';
+import {graphql, createRefetchContainer, RelayRefetchProp} from 'react-relay';
+import SearchTableLayout from 'components/SearchTableLayout';
+import PaginationBar from 'components/PaginationBar';
+import {CertificationRequestsContainer_query} from '__generated__/CertificationRequestsContainer_query.graphql';
+
+const TIME_ZONE = 'America/Vancouver';
```
[the remark this answers is not in the document]
Comment (dleard) [reply app/containers/Certifier/CertificationRequestsContainer.tsx:10]: https://youtrack.button.is/issue/GGIRCS-1553

File: app/components/PaginationBar.tsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import {Pagination} from 'react-bootstrap';
+
+interface Props {
+  setOffset: (...args: number[]) => void;
+  setActivePage: (...args: number[]) => void;
```
Comment (matthieu-foucault) [inline app/components/PaginationBar.tsx:6]:
```suggestion
  setOffset: (offset: number) => void;
  setActivePage: (page: number) => void;
```

File: app/pages/certifier/requests.tsx [hunk 5]
```diff
@@ -36,7 +59,14 @@ export default class CertifierRequests extends Component<Props> {
         session={query.session}
         allowedGroups={ALLOWED_GROUPS}
       >
-        <CertificationRequestsTable query={query.session.ciipUserBySub} />
+        {/* <CertificationRequestsTable query={query.session.ciipUserBySub} /> */}
```
[the remark this answers is not in the document]
Comment (matthieu-foucault) [reply app/pages/certifier/requests.tsx:61]: @dleard one leftover comment here

**Pull request merged**